pwiz_tools/Shared/ProteomeDb/Fasta/WebEnabledFastaImporter.cs

```diff
@@ -1313,7 +1313,7 @@ private int DoWebserviceLookup(IList<ProteinSearchInfo> proteins, char searchTyp
                 {
                     const string STATUS_REVIEWED = "reviewed"; // Uniprot reviewed status  // Not L10N
                     // now see if responses are ambiguous or not
-                    if (proteins.Count() == 1)

```

Comment:
Why are you allowed to do this?  So confusing...

Reply:
The Count method is an extension method for classes implementing IEnumerable, but since "proteins" is an IList it has a Count property that doesn't require enumeration of all items, instead it just keeps track of its size.

(If you look at the decompiled Count method you will notice that it actually tries to cast the IEnumerable to an IList to avoid enumerating all items, so there's not really a performance improvement)